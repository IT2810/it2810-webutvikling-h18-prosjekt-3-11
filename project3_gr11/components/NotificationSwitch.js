//Switch for enabling notification for daily reminder
import React, { Component } from 'react';
import { Switch, View, Text } from 'react-native';

export class NotificationSwitch extends Component {
    render() {
        return (
            <View>
                <Text>Enable daily reminder</Text>
            <Switch
                onValueChange={this.props.toggleNotif}
                value={this.props.notifValue}
            />
            </View>
        )
    }
}

