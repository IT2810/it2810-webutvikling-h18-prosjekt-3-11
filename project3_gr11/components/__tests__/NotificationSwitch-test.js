import React from 'react';
import renderer from 'react-test-renderer';
import { NotificationSwitch } from "../NotificationSwitch";

test('renders correctly', () => {
    const tree = renderer.create(<NotificationSwitch/>).toJSON();
    expect(tree).toMatchSnapshot();
});