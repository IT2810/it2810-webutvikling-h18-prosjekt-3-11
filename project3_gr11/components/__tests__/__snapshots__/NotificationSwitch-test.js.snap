// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Enable daily reminder
  </Text>
  <RCTSwitch
    disabled={false}
    onChange={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "height": 31,
          "width": 51,
        },
        undefined,
      ]
    }
    value={false}
  />
</View>
`;
